src/fsharp/est.fs

```diff
@@ -17,6 +16,12 @@ module internal ExtensionTyping =
     open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library // frontAndBack
     open Internal.Utilities.FileSystem
 
+#if FX_RESHAPED_REFLECTION
+    open Microsoft.FSharp.Core.ReflectionAdapters
+#else
+    type BindingFlags = System.Reflection.BindingFlags
```

Comment:
open System.Reflection caused some ambiguity at compile time across build types, I could get coreclr right or not coreclr right but not both.  It was actually easier to not open it, and be explicit.  I can take another look, but I remember struggling with this for quite a while.